### DISCUSSION TITLE
Neo4j package

### DISCUSSION THREAD

**Discussion by PreetamGKatakali** [langfuse/langfuse#4566] [Support]
Hi,i just wanted to ask i am using the neo4j package (from langchain_community.graphs import Neo4jGraph) when i build the component it is giving me the pip install neo4j where can i do that ?

**Comment by marcklingen** [comment]
Can you provide more context? I would love to help, but I don't understand the question.

**Comment by PreetamGKatakali** [comment]
using the docker image to run the lang flow (langflowai/langflow-nightly:latest) and in the custom component we are using :

from langchain_community.graphs import Neo4jGraph
 
kg = Neo4jGraph(
    url=NEO4J_URI, username=NEO4J_USERNAME, password=NEO4J_PASSWORD, database=NEO4J_DATABASE
)
 
when i build the component getting error as pip install neo4j

**Reply by marcklingen** [reply]
seems unrelated to langfuse, have you tried to install neo4j?
